### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by skhaz** [biomejs/pre-commit#14]
It used to work, but for the past few days, on two different MacBooks, it started showing the same error below:

```
[INFO] Installing environment for https://github.com/biomejs/pre-commit.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/Users/me/.cache/pre-commit/repoo6v7rpql/node_env-default/bin/node', '/Users/me/.cache/pre-commit/repoo6v7rpql/node_env-default/bin/npm', 'pack')
return code: 254
stdout: (none)
stderr:
    (node:28367) ExperimentalWarning: Support for loading ES Module in require() is an experimental feature and might change at any time
    (Use `node --trace-warnings ...` to show where the warning was created)
    npm warn tarball tarball data for file:/Users/me/.cache/pre-commit/repoo6v7rpql/ (null) seems to be corrupted. Trying again.
    npm warn tarball tarball data for file:/Users/me/.cache/pre-commit/repoo6v7rpql/ (null) seems to be corrupted. Trying again.
    npm error code ENOENT
    npm error syscall lstat
    npm error path /Users/me/.cache/pre-commit/repoo6v7rpql/ode_env-default/lib/node_modules/npm/node_modules/sprintf-js/dist/.gitattributes
    npm error errno -2
    npm error enoent ENOENT: no such file or directory, lstat '/Users/me/.cache/pre-commit/repoo6v7rpql/ode_env-default/lib/node_modules/npm/node_modules/sprintf-js/dist/.gitattributes'
    npm error enoent This is related to npm not being able to find a file.
    npm error enoent
    npm error A complete log of this run can be found in: /Users/me/.npm/_logs/2024-10-23T09_20_35_727Z-debug-0.log
Check the log at /Users/me/.cache/pre-commit/pre-commit.log
```

```
cat /Users/me/.npm/_logs/2024-10-23T09_20_35_727Z-debug-0.log
0 verbose cli /Users/me/.cache/pre-commit/repoo6v7rpql/node_env-default/bin/node /Users/me/.cache/pre-commit/repoo6v7rpql/node_env-default/bin/npm
1 info using npm@10.9.0
2 info using node@v23.0.0
3 silly config load:file:/Users/me/.cache/pre-commit/repoo6v7rpql/node_env-default/lib/node_modules/npm/npmrc
4 silly config load:file:/Users/me/.cache/pre-commit/repoo6v7rpql/.npmrc
5 silly config load:file:/Users/me/.npmrc
6 silly config load:file:/Users/me/.cache/pre-commit/repoo6v7rpql/node_env-default/etc/npmrc
7 verbose title npm pack
8 verbose argv "pack"
9 verbose logfile logs-max:10 dir:/Users/me/.npm/_logs/2024-10-23T09_20_35_727Z-
10 verbose logfile /Users/me/.npm/_logs/2024-10-23T09_20_35_727Z-debug-0.log
11 silly logfile done cleaning log files
12 silly packumentCache heap:4345298944 maxSize:1086324736 maxEntrySize:543162368
13 warn tarball tarball data for file:/Users/me/.cache/pre-commit/repoo6v7rpql/ (null) seems to be corrupted. Trying again.
14 warn tarball tarball data for file:/Users/me/.cache/pre-commit/repoo6v7rpql/ (null) seems to be corrupted. Trying again.
15 verbose stack Error: ENOENT: no such file or directory, lstat '/Users/me/.cache/pre-commit/repoo6v7rpql/ode_env-default/lib/node_modules/npm/node_modules/sprintf-js/dist/.gitattributes'
16 error code ENOENT
17 error syscall lstat
18 error path /Users/me/.cache/pre-commit/repoo6v7rpql/ode_env-default/lib/node_modules/npm/node_modules/sprintf-js/dist/.gitattributes
19 error errno -2
20 error enoent ENOENT: no such file or directory, lstat '/Users/me/.cache/pre-commit/repoo6v7rpql/ode_env-default/lib/node_modules/npm/node_modules/sprintf-js/dist/.gitattributes'
21 error enoent This is related to npm not being able to find a file.
21 error enoent
22 verbose cwd /Users/me/.cache/pre-commit/repoo6v7rpql
23 verbose os Darwin 24.0.0
24 verbose node v23.0.0
25 verbose npm  v10.9.0
26 verbose exit -2
27 verbose code -2
```

**Comment by ematipico** [comment]
Please do a quick search next time.

Duplicate of https://github.com/biomejs/pre-commit/issues/13

Closing

**Comment by skhaz** [comment]
Please don’t be arrogant with your users. I checked the issues, and there were none open, so I assumed no one had encountered the problem before.

And you probably know better than I do that GitHub’s search functionality isn’t that great.

**Comment by ematipico** [comment]
Apologies 😔 

I wanted to come up as an arrogant
